backend/Tasque.BLL/Services/EntityCrudService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Tasque.Core.Common.Entities.Abstract;
+using Tasque.Core.DAL;
+
+namespace Tasque.Core.BLL.Services
+{
+    public abstract class EntityCrudService<TEntity> : IEntityCrudService<TEntity> where TEntity : BaseEntity
+    {
+        private readonly DataContext _db;
+        public EntityCrudService(DataContext db)
+        {
+            _db = db;
+        }
+
+        public virtual TEntity Create(TEntity entity)
+        {
+            _db.Set<TEntity>().Add(entity);
+            _db.SaveChanges();
+            return entity;
+        }
+
+        public TEntity? GetOne(int id)
```

Comment:
To better understanding I suggest to name methods as "GetById"

Reply:
Check [this](https://github.com/BinaryStudioAcademy/bsa-2022-tasque/pull/21#discussion_r942213724) conversation

Reply:
Imo it's fine, and we can change it later if needed

Reply:
Renamed